### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by oysteinsigholt**
Tests revealed a bug in `index.js:70` that has been fixed in this commit.

### CODE REVIEW

**Comment by oysteinsigholt** [comment]
Note that `should not match invalid Open API specs` fails because its feature is not yet implemented.

**Comment by andereld** [comment]
If a test fails because the feature is not yet implemented, it should be marked as pending using one of the following:

https://mochajs.org/#pending-tests
https://mochajs.org/#inclusive-tests

That way, you can add the tests/specs, but not break the build.

File: test/test.js [hunk 1]
```diff
@@ -1,11 +1,111 @@
 const Metalsmith = require('metalsmith');
-const chai = require('chai');
+const expect = require('expect');
 const plugin = require('./../index.js');
```
Comment (andereld) [inline test/test.js]: `plugin` should probably be called `metalsmithSwaggerUi` or something. I was confused by `.use(plugin())` in `before()`.

File: test/test.js [hunk 2]
```diff
@@ -1,11 +1,111 @@
...
+    if (err) done(err);
+    done();
+  });
+});
+
+describe('metalsmith-swaggerui', ()=> {
   describe('#isOpenAPI',  () => {
-    it('should match json files',  () => {
-      //TODO
+    it('should match .json files',  () => {
```
Comment (andereld) [inline test/test.js]: Is there a particular reason you have two spaces rather than one before the callback function?

**Comment by oysteinsigholt** [comment]
The failing test has been marked as pending, the double spaces were unintentional. Code should be properly linted now. I also moved the `src` folders contents to `assets/fixtures`

**Pull request merged**